apps/native-component-list/src/screens/Reanimated/ReanimatedSwipeable.tsx

```diff
@@ -64,7 +64,11 @@ function SwipableList(): React.ReactElement {
     <View style={s.container}>
       <FlatList
         data={data}
+        // @ts-expect-error: regression in RNGH typings?
+        // Type 'unknown' is not assignable to type 'Data'.ts(2322)
```

Comment:
cc @m-bert

Reply:
I'm pretty sure it happened because of [this PR](https://github.com/software-mansion/react-native-gesture-handler/pull/2835) which will be reverted soon.